prow/config/config.go

```diff
@@ -37,6 +37,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation"
 	"sigs.k8s.io/yaml"
 
+	buildv1alpha1 "github.com/knative/build/pkg/apis/build/v1alpha1"
 	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
 	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
```

Comment:
why do we import twice?

Reply:
lol because rebase is dumb? /shrug will fix

Reply:
lol, actually I blame @stevekuznetsov

Reply:
blame tkaen